tests/integration-tests/test_runner.py

```diff
@@ -305,6 +306,11 @@ def _init_argparser():
         help="S3 URI of the ParallelCluster API spec",
         default=TEST_DEFAULTS.get("public_ecr_image_uri"),
     )
+    debug_group.add_argument(
```

Comment:
can we have an api_group and we move there all api related options?

Reply:
:+1: done.